ops/pebble.py

```diff
@@ -1461,20 +1485,28 @@ def from_dict(cls, d: _CheckInfoDict) -> CheckInfo:
             status = CheckStatus(d['status'])
         except ValueError:
             status = d['status']
+        change_id = ChangeID(d['change-id']) if 'change-id' in d else None
```

Comment:
Perhaps:

```suggestion
        change_id = ChangeID(d.get('change-id'))
```

Reply:
That would result in `ChangeID(None)` if `change-id` wasn't present in the dictionary, which won't work, because `ChangeID` is a string subclass.

Reply:
Ah! sorry that somehow escaped my attention.